### PR TITLE
Update config files and enable docker toggle

### PR DESCRIPTION
**Pull request by predominant**
This updates the config generation to match the new format from Drone.io, and also allows a toggle to disable installing docker on the same node (good for production deployments).

Quick code review would be great. Suggest we bump this to 0.6.0 for release.

### CODE REVIEW

**Comment by justincampbell** [comment]
@predominant looks good! I've pushed a fix for the Rubocop failure to master, if you could rebase

File: attributes/default.rb [hunk 1]
```diff
@@ -10,8 +11,5 @@
   'database' => {
     'driver' => 'sqlite3',
     'datasource' => '/var/lib/drone/drone.sqlite'
-  },
-  'registration' => {
-    'open' => true
```
Comment (justincampbell) [inline attributes/default.rb]: Why was this removed? Has it been removed from Drone as well?

Comment (predominant) [inline attributes/default.rb]: Yeah, registration option now goes into each source control type. The readme was updated to show examples.

**Comment by predominant** [comment]
Rebase was done. Still failing on Travis for Ruby 1.9.3 due to Ohai requiring 2.0+ But should be fine to merge.

**Comment by predominant** [comment]
Merging.
